apps/driving_pi/Makefile

```diff
@@ -0,0 +1,139 @@
+NCCOMPILE = mo.py
+
+GREEN = '\033[1;32m'
+YELLOW = '\033[1;33m'
+RED = '\033[1;31m'
+NOCOLOR = '\033[0m'
+
+ZOO_RELATIVE_DIR = ../../omz
+MODEL_DOWNLOADER_DIR = open_model_zoo/tools/downloader
+MODEL_DOWNLOADER_FILENAME = downloader.py
+ROAD_SEG_MODEL_ZOO_DIR = Transportation/segmentation/curbs/dldt/FP16
+ROAD_SEG_MODEL_FILE_NAME_BASE = road-segmentation-adas-0001
+ROAD_SEG_DESTINATION_DIR = src/data/road-segmentation
```

Comment:
This directory (data) does not exist in #319, #320, or #321 nor is it created - make run fails without it because there is no place to copy the IRs to.